examples/rigid/ik_shadow_hand.py

```diff
@@ -66,24 +66,27 @@ def main():
     target_quat = np.array([1, 0, 0, 0])
     index_finger_distal = robot.get_link("index_finger_distal")
     middle_finger_distal = robot.get_link("middle_finger_distal")
-    forearm = robot.get_link("forearm")
+    wrist = robot.get_link("wrist")  # there is no link named "forearm"
 
     center = np.array([0.5, 0.5, 0.2])
     r1 = 0.1
     r2 = 0.13
+    # FIXME: we have this Debug info and the simulation runs very slow
```

Comment:
do you know how to fix this non-continuous memory? Although it's not a breaking bug @duburcqa